front/src/applications/stdcmV2/components/StdcmResults/StdcmResultsTable.tsx

```diff
@@ -48,7 +52,10 @@ const StcdmResultsTable = ({
             const isFirstStep = index === 0;
             const isLastStep = index === operationalPointsList.length - 1;
             const prevStep = operationalPointsList[index - 1];
-            const shouldRenderRow = isFirstStep || step.duration > 0 || isLastStep;
+            const isRequestedPathStep = pathSteps.find(
+              (pathStep) => pathStep && pathStep.name === step.name && pathStep.ch === step.ch
+            );
```

Comment:
```suggestion
            const isRequestedPathStep = stdcmData.simulationPathSteps.some(
              (pathStep) => pathStep && pathStep.name === step.name && pathStep.ch === step.ch
            );
```

I think we don't need to import useSelector, since we already have access to the simulation path steps in stdcmData.